scaffold/github/actions/common/set-env/action.yml

```diff
@@ -12,6 +12,15 @@ runs:
   steps:
     - run: |
         echo "Setting Drainpipe environment variables:"
-        echo "DRAINPIPE_PR_NUMBER=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')" >> $GITHUB_ENV
-        echo "DRAINPIPE_SHA=$(cat $GITHUB_EVENT_PATH | jq -r .pull_request.head.sha)" >> $GITHUB_ENV
+
```

Comment:
I will try to revert this one after/if all tests pass now.

Reply:
Reverted. I will try a deploy to DEV on a real site to see if deploying still works

Reply:
The deployment didn´t work when I restored the file.
The changes in this PR are needed.